integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/NewCarbonDataLoadRDD.scala

```diff
@@ -51,7 +51,6 @@ class NewCarbonDataLoadRDD[K, V](
     sc: SparkContext,
```

Comment:
I think we can try it later. Currently because of newTaskAttemptID and some other function signature is changed in spark2, so need to duplicate code temporarily